### ISSUE TITLE
Cannot read properties of undefined (reading 'filter')

### ISSUE THREAD

**Issue by altonotch** [Noovolari/leapp#567]
**Describe the bug**
When going to SSM, selecting a region, see a loading sign, then instead of a list of servers i get this error: `Cannot read properties of undefined (reading 'filter')`.

**Leapp Version**
Version 0.26.1 (0.26.1)

**Expected behavior**
Either a clear message of what's missing or the actual list of servers to connect to.

**Desktop:**
 - OS: Mac OSx
 - OS Version 15.0.1 (24A348)
 - Leapp Version 0.26.1

**Additional context**
I did some disk clearing in Caches, maybe this is related? Also, upgraded to Sequoia. If you can direct me to how to debug this issue, would love to give it a try as well.

**Comment by ericvilla** [comment]
Hi @altonotch, thanks for reaching out! You can analyse your logs as described in the [docs](https://docs.leapp.cloud/latest/troubleshooting/app-data/#logs-file). Let me check the logic that lists the instances accessible through SSM

**Comment by ericvilla** [comment]
@altonotch I probably found the root cause. Please, check if there are any EC2 instances without Name tag; I suggest to tag them and retry to list SSM instances from Leapp

**Comment by altonotch** [comment]
> @altonotch I probably found the root cause. Please, check if there are any EC2 instances without Name tag; I suggest to tag them and retry to list SSM instances from Leapp

@ericvilla You're absolutely right! A few instances were missing Name tag. When all instances have the tag, SSM in Leapp works as expected. I think I saw it in the past falling back to instance ID 🤔

**Comment by ericvilla** [comment]
@altonotch I'm glad it solved the issue! I don't remember if Leapp was falling back to the instance ID; it could be a good solution
